keps/sig-api-machinery/1965-kube-apiserver-identity/README.md

```diff
@@ -179,7 +179,7 @@ Alpha should provide basic functionality covered with tests described above.
 
 #### Beta -> GA Graduation
 
-==TODO==
+  - support for aggregated apiservers
```

Comment:
@enj last I checked this was the biggest thing that was left for GA, anything else come to mind?